### ISSUE TITLE
consistent-type-export -> consistent-type-exports

### ISSUE THREAD

**Issue by NikitaIT** [oxc-project/oxc#2063]
Incorrect:
```sh
>   ⚠ typescript-eslint(consistent-type-export): Consistent type exports
> oxlint -D all -A consistent-type-export 
> oops!
```

Correct:
```sh
> oxlint --rules
• typescript: consistent-type-exports
> oxlint -D all -A consistent-type-exports 
```

**Comment by Boshen** [comment]
Good catch, thank you.
